src/components/search-box/SearchBox.jsx

```diff
@@ -31,10 +32,30 @@ const Container = styled.form`
 `;
 
 const SearchBox = (props) => {
+  const [searchValue, setSearchValue] = useState('');
```

Comment:
Correct me if I'm wrong but this seems to be the same value conceptually as `searchValue` in `App.jsx`.  If this is the case then I recommend removing this `useState` and passing the ones from `App` as props using `Header`.  In the State Management lesson we will learn alternative ways to this but that would be better than repeating state as it currently is

Reply:
Feel free to ask questions or message me on slack for a re-review of this

Reply:
Of course, you are right, i didn't notice to "lift state up" from SearchBox to App via Header. Likewise, in the next Mini-Challenge we will improve this feature using Contex API. Thank you.